### ISSUE TITLE
Any plans for continue supporting library?

### ISSUE THREAD

**Issue by pushnovn** [linnobita/rulerjs-libs#1]
Hello!

Have you got any plans to continue supporting this project?
Complete TODOs, add new features according to the api?

**Comment by linnobita** [comment]
Yes. I will continue supporting this project. But too busy with works these days. I will continue to work on it whenever I've got time. Thanks for being interested to this library!
